clusterloader2/pkg/measurement/measurementfactory.go

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
Since you're already inside measurement/ directory, I think you can simply name this file `factory.go` or sth.

Reply:
done